cibuildwheel/linux.py

```diff
@@ -88,6 +88,12 @@ def build(project_dir, output_dir, test_command, before_test, test_requires, tes
         ('pp', 'manylinux_x86_64', manylinux_images['pypy_x86_64']),
     ]
 
+    abs_project_dir = os.path.abspath(project_dir)
+    abs_package_dir = os.path.abspath(package_dir)
+
+    container_project_dir = '/project'
+    container_package_dir = os.path.join(container_project_dir, os.path.relpath(abs_package_dir, os.path.commonprefix([abs_project_dir, abs_package_dir]))),
```

Comment:
Not if package_dir was passed as absolute path, I guess? Or, in that case we'd need to specify it needs to be a relative path?

Reply:
ah yeah, good point. Then the relpath step would still be needed, but it could be left relative to the working dir and used like that inside the container.